scripts/newinstall.sh

```diff
@@ -306,11 +272,11 @@ if true; then
 		CMD_SETUP_GIT='setup git'
 	fi
 
-	if [[ $WITH_ANACONDA = 1 ]]; then
-		echo "Installing Anaconda Python Distribution ... "
-		$cmd eups distrib install --repository="$EUPS_PKGROOT" anaconda
-		$cmd setup anaconda
-		CMD_SETUP_ANACONDA='setup anaconda'
+	if [[ $WITH_MINICONDA2 = 1 ]]; then
+		echo "Installing Miniconda2 Python Distribution ... "
+		$cmd eups distrib install --repository="$EUPS_PKGROOT" miniconda "$MINICONDA2_VERSION"
+		$cmd setup miniconda
+		CMD_SETUP_MINICONDA2='setup miniconda'
```

Comment:
Why are we installing `miniconda` everywhere but talking about `miniconda2` in all the commentary?